docs/core_docs/docs/integrations/chat/google_vertex_ai.mdx

```diff
@@ -125,7 +125,7 @@ See the LangSmith trace for the example above [here](https://smith.langchain.com
 The Gemini family of models not only support tool calling, but can also be used in the OpenAI Tools agent.
 Here's an example:
 
-import AgentsExample from "@examples/models/chat/chat_mistralai_agents.ts";
+import AgentsExample from "@examples/models/chat/chat_vertexai_agents.ts";
```

Comment:
double check. vertexai?

Reply:
This change was from original `langchainjs`'s commit, I think it can be ignored